10up/distributor
# Pull contents via ajax

### Description of the Change
Now it pulls contents via ajax to avoid potential request timeout and enhanced user experience.


Closes #121 

### How to test the Change

- Go to pull page
- Pull multiple or single post
- You should see a modal with minimal progress information

### Changelog Entry

> Changed - Pull mechanism updated to ajax workflow


### Credits

Props @jayedul 


### Checklist:
